cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/statements/expressions/CallExpression.kt

```diff
@@ -87,12 +87,18 @@ open class CallExpression : Expression(), HasType.TypeObserver, ArgumentHolder {
      */
     var arguments by PropertyEdgeDelegate(CallExpression::argumentEdges)
 
+    /** The list of argument types (aka the signature). */
+    val signature: List<Type>
+        get() {
+            return argumentEdges.map { it.end.type }
+        }
+
     /**
      * The expression that is being "called". This is currently not yet used in the [SymbolResolver]
      * but will be in the future. In most cases, this is a [Reference] and its [Reference.refersTo]
      * is intentionally left empty. It is not filled by the [SymbolResolver].
      */
-    @AST var callee: Expression? = null
+    @AST var callee: Expression = ProblemExpression("could not parse callee")
```

Comment:
Is it a problem of parsing or translation? I think, we should set the `ProblemNode.ProblemType` to something different (currently it's `TRANSLATION` by default). Might require another option in the enum

Reply:
It should be set by the frontend so I guess `TRANSLATION` is ok.